apps/src/code-studio/components/ExportAllowedDialog.jsx

```diff
@@ -0,0 +1,743 @@
+            Android
+          </p>
+        </div>
+        <div style={styles.section}>
+          <p style={styles.p}>
+            We will create a mobile app that reflects the current state of your
+            Code Studio creation. Then, you can install the app on your mobile
+            device.
+          </p>
+          {/*<p style={styles.p}>
+            The first step is to install the Expo app on your mobile device so
+            you can test your project within the Expo app.
+          </p>
+          <button
+            type="button"
+            style={styles.iosAppStoreButton}
+            onClick={this.onInstallExpoIOS}
+          >
+            iOS Expo App
+          </button>
+          <button
+            type="button"
+            style={styles.androidGooglePlayButton}
+            onClick={this.onInstallExpoAndroid}
+          >
+            Android Expo App
+          </button>*/}
+        </div>
+      </div>
+    );
+  }
+
+  renderExportPage() {
+    const {
+      exporting,
+      exportError,
+      expoUri,
+      expoSnackId,
+      iconUri,
+      splashImageUri
+    } = this.state;
+    // TODO: This page should be updated as a transition page to snack.expo.io
+    // when iOS IPA export is ready to go
+    return (
+      <div>
+        <div style={styles.section}>
+          <p style={styles.title}>Preview your project in the Expo app</p>
+        </div>
+        <div style={styles.section}>
+          <p style={styles.p}>TBD.</p>
+          <p style={styles.p}>{`exporting: ${exporting}`}</p>
+          <p style={styles.p}>{`exportError: ${exportError}`}</p>
+          <p style={styles.p}>{`expoUri: ${expoUri}`}</p>
+          <p style={styles.p}>{`expoSnackId: ${expoSnackId}`}</p>
+          <p style={styles.p}>{`iconUri: ${iconUri}`}</p>
+          <p style={styles.p}>{`splashImageUri: ${splashImageUri}`}</p>
+        </div>
+      </div>
+    );
+  }
+
+  renderPlatformPage() {
+    return (
+      <div>
+        <div style={styles.section}>
+          <p style={styles.title}>Choose your platform</p>
+        </div>
+        <div style={styles.section}>
+          <div>
+            <input
+              style={styles.radioInput}
+              type="radio"
+              id="radioAndroid"
+              readOnly
+              checked
+            />
+            <label htmlFor="radioAndroid" style={styles.radioLabel}>
+              I have an Android device
+            </label>
+          </div>
+          <div>
+            <input
+              style={styles.radioInput}
+              type="radio"
+              id="radioIOS"
+              disabled
+            />
+            <label htmlFor="radioIOS" style={styles.radioLabelDisabled}>
+              I have an iOS device (currently not supported)
+            </label>
+          </div>
+        </div>
+        <div style={styles.section}>
+          <p style={styles.p}>
+            <b>Note: </b>Exporting will take 10-15 minutes. Any changes made
```

Comment:
"Note: Exporting will take 10-15 minutes. If you change your app after you start exporting, those changes will not be included."